api/app/routers/fire_watch.py

```diff
@@ -167,4 +174,19 @@ async def save_new_fire_watch(fire_watch_input: FireWatchInput, token=Depends(au
         new_fire_watch_id = await save_fire_watch(session, db_fire_watch)
         new_fire_watch = await get_fire_watch_by_id(session, new_fire_watch_id)
         fire_watch_output = marshall_fire_watch_db_to_api(new_fire_watch)
-        return FireWatchResponse(fire_watch=fire_watch_output)
\ No newline at end of file
+        return FireWatchResponse(fire_watch=fire_watch_output)
+
+
+@router.get("/weather/{start_date}/{end_date}", response_model=FireWatchWeatherResponse)
```

Comment:
Should this be a POST since we're passing a list of stations in the request body?